backend/person-service/src/main/kotlin/com/nosql/personservice/service/impl/DefaultPersonService.kt

```diff
@@ -0,0 +1,28 @@
+package com.nosql.personservice.service.impl
+
+import com.nosql.personservice.component.impl.DefaultPersonComponent
+import com.nosql.personservice.converter.PersonDtoToPersonEntityConverter
+import com.nosql.personservice.dto.DefaultApiResponseDto
+import com.nosql.personservice.dto.PersonDto
+import com.nosql.personservice.service.PersonService
+import org.bson.types.ObjectId
+import org.springframework.stereotype.Service
+
+@Service
+class DefaultPersonService(
+    private val personComponent: DefaultPersonComponent,
+    private val personDtoToPersonEntityConverter: PersonDtoToPersonEntityConverter,
```

Comment:
use the conversion service and the `ConversionServiceUtil.kt` in `auth-service`

Reply:
Fixed, but we can have troubles)))))